datamol/mol.py

```diff
@@ -1376,3 +1384,42 @@ def get_atom_positions(
         positions = positions[mapped_indices, :]
 
     return positions
+
+
+def remove_salts(mol: Mol, remover: Optional[SaltRemover] = SALT_REMOVER) -> Mol:
+    """Remove all salts from the molecule
+
+    Args:
+        mol: A molecule.
+        remover: A object that defines salts to be removed.
+
+    See Also:
+        <rdkit.Chem.SaltRemover.SaltRemover>
+    """
+    return remover.StripMol(mol)
```

Comment:
Since I never really know whether rdkit is going to modify the input molecule, I usually always prefer to add a `dm.copy()` on the input molecule to be safe.

Reply:
In strip mol, I think it would be worth it to control the default argument. For example, what happens if your molecule is a solvent by itself ? We should consider the case of people just blindly applying processing on a full dataset

Reply:
ah good point. should we catch None and throw an error? maybe put that behind a boolean flag?

Reply:
Maybe check the default behavior of StripMol when you only input a solvent?

Reply:
I think forwarding the argument of StripMol to the argument of the function would be a good idea:

https://www.rdkit.org/docs/source/rdkit.Chem.SaltRemover.html

Reply:
In the case the input is a solvent itself, the solvent will be removed based on the predefined files. In that case, one need to use `dontRemoveEverthing` and/or defined salts data themselves. 
So keeping arguments and the option of customized salts should be flexible enough.